gatorgrade/main.py

```diff
@@ -31,6 +36,7 @@ def generate(
 ):
     """Generate a gatorgrade.yml file."""
     if force is False:
+        checks = generate_config(FILE)
```

Comment:
Unfortunately, `generate_config` has not been implemented yet. However, based on discussions I've had with the Generate team, the argument to the `generate_config` will likely not be the path to the `gatorgrade.yml` file. You should continue to communicate with the Generate team about the progress of the `generate_config` command!